functions/Get-DbaDbRoleMember.ps1

```diff
@@ -106,6 +106,17 @@ function Get-DbaDbRoleMember {
                 Stop-Function -Message 'Failure' -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
             }
 
+            foreach ($item in $Database) {
+                Write-Message -Level Verbose -Message "Check if database: $item on $instance is accessible or not"
+                try {
```

Comment:
the try catch will not be required here, unless I'm missing something. 

this could be done like so:

```
if (-not $server.Databases[$item].IsAccessible) {
   Stop-Function -Message Failure -Category ResourceUnavailable -Target $instance -Continue
}
```

Reply:
It's just my habit to use try/catch and throw. I'll make adjustments soon